src/transformers/pipelines.py

```diff
@@ -974,11 +1002,30 @@ def __call__(self, *args, **kwargs):
         """
         outputs = super().__call__(*args, **kwargs)
 
-        if self.model.config.num_labels == 1:
-            scores = 1.0 / (1.0 + np.exp(-outputs))
+        return_all_scores = return_all_scores if return_all_scores is not None else self.return_all_scores
+        function_to_apply = function_to_apply if function_to_apply is not None else self.function_to_apply
```

Comment:
Per @mfuntowicz's request, the arguments can now be handled via the `__call__` method of the pipeline, as well as the `__init__` and the model configuration.

Reply:
Minor NIT: I find the following slightly more readable. Feel free to ignore if you don't agree.
```python
if return_all_scores is None:
    return_all_score = self.return_all_scores

if function_to_apply is None:
   function_to_apply = self.function_to_apply
```